docs/can-guides/commitment/recipes/credit-card-advanced/1-setup.html

```diff
@@ -1,20 +1,6 @@
 <!doctype html>
 <title>Credit Card Guide (Advanced)</title>
 <body>
-<script src="https://unpkg.com/kefir@3/dist/kefir.min.js"></script>
-<script src="https://unpkg.com/can@4/dist/global/can.all.js"></script>
 <script src="https://js.stripe.com/v2/"></script>
```

Comment:
It doesn’t look like Stripe is used in this guide. I would remove all these lines.

Reply:
Fixed